src/pykeen/models/uncertainty.py

```diff
@@ -14,7 +24,7 @@
     "predict_h_uncertain",
     "predict_t_uncertain",
     "predict_r_uncertain",
-    "predict_uncertain_helper",
+    # "predict_uncertain_helper", # TODO: this is a helper method; should we make it public?
```

Comment:
I did this because it had lots of nice documentation I thought might be useful for people to see